virttest/utils_stress.py

```diff
@@ -0,0 +1,65 @@
+import random
+import time
+
+from avocado.utils import cpu
+
+from virttest import virsh
+from virttest import utils_hotplug
+from virttest.utils_test import libvirt
+
+
+def vm_stress_events(event, vm, params):
```

Comment:
we can have stress associated with component like cpu, lifecycle, memory (todo) separately, based on the params set it in __init__ and trigger it.

Reply:
it is better to have freeflow method like this and choose the events we want in testcase, we need not restrict components here, rather we can mix and match many events in testcase, what say?

Reply:
mix and match would work better if we segregate,
- instead of calling each time, we can set it once and trigger it once
- redundancy can be avoided
```python
class Stress_Events():
    def __init__(self, vm):
        self.vm = vm
        self.events = self.vm.params.get("events", ["reboot"])
        self.ignore_status = self.vm.params.get("event_ignore_status", "no") == "yes"

    def check_status(self):
        libvirt.check_exit_status(self.ret)

    def trigger(self):
        dargs = self.vm.params.get("dargs", None)
        args = {'name':self.vm.name, **dargs}
        for each_event in events:
            self.event(each_event, **args)

    def event(self, event_name, args):
        if virsh.__dict__.get(event_name):
            self.ret = virsh.event_name(args)
        elif self.vm.__dict__.get(event_name):
            self.vm.event_name(args)
        else:
            # handle appropriately
        if not self.ignore_status:
            self.check_status()

```

Reply:
@balamuruhans yes, partly we can go with this for init , but the events has to be run in parallel using `threading`, let me see if I can introduce `threading` as well here, or I can use the class for common init.